### PR TITLE
ci: Add tests for database on web

### PR DESCRIPTION
**Pull request by krille-chan**

### CODE REVIEW

**Comment by td-famedly** [comment]
You probably also want to make `dart_web_compatible` and `database_web_tests` both required with this PR?

File: lib/src/database/indexeddb_box.dart [hunk 1]
```diff
@@ -13,6 +13,8 @@ class BoxCollection with ZoneTransactionMixin {
 
   BoxCollection(this._db, this.boxNames, this.name);
 
+  static const bool isWeb = true;
```
Comment (td-famedly) [inline lib/src/database/indexeddb_box.dart:16]: Could the isWeb checks be replaced by the kIsWeb implementation? if they run on a chromedriver I assume kisWeb should be true. We already do this in some voip code.

https://api.flutter.dev/flutter/foundation/kIsWeb-constant.html

Comment (td-famedly) [reply lib/src/database/indexeddb_box.dart:16]: no but you can just do `bool.fromEnvironment('dart.library.js_util');`

Comment (krille-chan) [reply lib/src/database/indexeddb_box.dart:16]: oh really? Wasn't aware of this. That's great! Thank you so much :)

Comment (krille-chan) [reply lib/src/database/indexeddb_box.dart]: Have changed it :)

**Review by td-famedly — APPROVED** [review]

**Pull request merged**